docs/internals/dev.rst

```diff
@@ -0,0 +1,104 @@
+=================
+Developing EdgeDB
+=================
+
+This section describes how to build EdgeDB locally, how to use its
+internal tools, and how to contribute to it.
+
+
+Building Locally
+================
+
+The following instructions should be used to create a "dev" build on
+Linux or macOS.  Windows is not currently supported.
+
```

Comment:
List the build requirements:

- GNU make version 3.80 or newer
- C compiler (GCC or clang)
- autotools
- Python 3.7 dev package
- Bison 1.875 or later
- Flex 2.5.31 or later
- Perl 5.8.3 or later